lua/wikis/dota2/MatchGroup/Input/Custom.lua

```diff
@@ -110,10 +110,16 @@ function MatchFunctions.getLinks(match, games)
 	links.stratz = {}
 	links.dotabuff = {}
 	links.datdota = {}
+	links.blasttvdota = {}
 
 	Array.forEach(
 		Array.filter(games, function(map) return map.matchid ~= nil end),
 		function(map, mapIndex)
+			-- only 1 matchId is needed for blasttv
+			-- blast.tv/dota/match/{matchid} redirects to series page with all matches
+			if mapIndex == 1 then
+				links.blasttvdota[mapIndex] = 'https://www.blast.tv/dota/match/' .. map.matchid
+			end
```

Comment:
imo do not do it per map/game
instead handle it on match level and for that you do not need this change at all, only the correct data change is needed